### DISCUSSION TITLE
Query about import

### DISCUSSION THREAD

**Discussion by simonmcnair** [beetbox/beets#4697] [General]
I'm importing a huge number of files in a mix of singletons, albums, soundtacks, OST etc from a variety of sources.  I don't expect Beets to get things right first time, or to even know what to do with my mess.

What I would like though is steady incremental improvement and lack of repetition.  If I have run a CRC on it, or badfiles I'd prefer not to run it again unless I want to.  OR at least see if the filenames match the tags.

In effect I'm after a staging ares/directories .  i.e. unsorted, partially collated, fit-for-purpose

Is my workflow wrong ?  Is there a way to achieve my end goal pls ?
Cheers

**Comment by JOJ0** [comment]
I think you should rephrase.

You ask about if your workflow is wrong, but you never described your workflow in the first place :-)

Hope that helps to form more concrete questions :-)
